pkg/syncer/syncers/downsyncer.go

```diff
@@ -274,6 +288,60 @@ func (ds *DownSyncer) SyncMany(resource edgev2alpha1.EdgeSyncConfigResource, con
 	return nil
 }
 
+// Compute resource object to be pushed from upstreamResource and downstreamResource
+//   - updatedResource is the computed resource object to be pushed
+//   - noDiff is true if the computed updatedResource is no difference from the downstream resource.
+func (ds *DownSyncer) computeUpdatedResource(upstreamResource *unstructured.Unstructured, downstreamResource *unstructured.Unstructured) (updatedResource *unstructured.Unstructured, noDiff bool) {
+	if !isDownsyncOverwrite(upstreamResource) {
+		ds.logger.V(2).Info(fmt.Sprintf("  downsync-overwrite of %q is marked as false", upstreamResource.GetName()))
+		annotations := downstreamResource.GetAnnotations()
+		value, ok := annotations[downsyncOverwriteKey]
+		if ok && value == "false" {
```

Comment:
Shouldn't the value in question here and on new line 304 be the value from upstreamResource, which this code only knows is neither "true" nor the empty string?